cli-plugins/apps/getting-started/configuration.hbs.md

```diff
@@ -0,0 +1,168 @@
+# Configuration
+
+## <a id='changing-clusters'></a>Changing clusters with --context
+
+The Apps CLI plug-in refers to the default kubeconfig file to access a Kubernetes cluster.
+When you run a `tanzu apps` command, the plug-in uses the default context that is defined in that
+kubeconfig file (located by default at `HOME/.kube/config`).
+
+There are two ways to change the target cluster:
+
+1. Use `kubectl config use-context CONTEXT-NAME` to change the default context. All subsequent
+`tanzu apps` commands target the cluster defined in the new default kubeconfig context.
+
+2. Include the `--context CONTEXT-NAME` flag when running any `tanzu apps` command.
+
+   >**Note** Any subsequent `tanzu apps` commands that do not include the `--context CONTENT-NAME`
+     flag continue to use the default context set in the kubeconfig.
+
+## <a id='override-kubeconfig'></a>Overriding the default kubeconfig
+
+There are two approaches to overriding the default kubeconfig:
+
+1. Set the environment variable `KUBECONFIG=PATH` to change the kubeconfig the Apps CLI plug-in will
+   reference.
+
+   All subsequent `tanzu apps` commands reference the non-default kubeconfig assigned to the environment
+   variable.
+
+2. Include the  `--kubeconfig path` flag when running any `tanzu apps` command.
+
+   >**Note** Any subsequent `tanzu apps` commands that do not include the `--context CONTEXT-NAME` flag
+   continue to use the default context set in the kubeconfig.
+
+For more information about kubeconfig, see 
+[Configure Access to Multiple Clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+in the Kubernetes documentation.
+
+## Suppressing color with --no-color flag
+
+Most of Tanzu Apps Plug-in commands have emojis and colored output with the intention to be more user-friendly.
+
+However, sometimes color, emojis and other characters are not needed (e.g. scripting) or even well 
+interpreted in certain terminals and the best way to suppress them is using the `--no-color` flag.
+
+So, for example, in a workload that is created through local path, which usually shows emojis and a
+progress bar, these special characters would be avoid by using `--no-color`.
+
+```bash
+tanzu apps workload apply my-workload --local-path path/to/my/source -s my-registry.ext/my-project/my-workload --type web --no-color
```

Comment:
Now that we've got LSP, let's keep things simple and remove the `-s...`
And generally speaking in our docs, we should use the full flag name rather than the aliases.

Reply:
changed